default/stringtables/en.txt

```diff
@@ -3194,6 +3194,20 @@ Unable to create persistent_config.xml, check log file for details.
 MAP_BTN_TURN_UPDATE
 Turn %1%
 
+MAP_BTN_TURN_TOOLTIP
+Start Turn
+
+MAP_BTN_TURN_TOOLTIP_DESC
+'''Starts turn processing with the following sequence:
```

Comment:
Consider observers, moderators, and when waiting for other players or the server...?

Reply:
Done except for moderators, does the turn button behave differently for them?